### DISCUSSION TITLE
PnP Search filters contains "GTset|.." as a refiner

### DISCUSSION THREAD

**Discussion by SiebeAmexio** [microsoft-search/pnp-modern-search#4037] [General]
We have a customer that has a lot of Teams and a page on the hub site that allows the users to search for documents in those Teams. The amount of documents every user sees depends on the permissions and the amount of Teams he's part of. Now we have a user that sees the values in the refiner from the termset "Tags" (like everybody) but also the Termset it self (see screenshot)

The refiner "Tags" is a taxonomy field and the "GTSet.." is the entire "Tags" termset. All values below it are terms of that termset. The site is only available in one language. With both Localization on and off , the user sees this "GTSet".

The user who experiences this issue is part of a lot of Teams and can find around 600.000 documents. I don't have this issue and I see around 5-6000 documents.

<img width="623" alt="image" src="https://github.com/user-attachments/assets/925f24a1-55d5-4346-b08f-2409ca32f626">

**Comment by kasperbolarsen** [comment]
Hi, I will need to know which version of PnP Modern Search you are using.
Try enabling Localization and let me know if it solved the issue

**Reply by SiebeAmexio** [reply]
Hi Kasper, Thank you for your quick reply! 

the version of PnP Modern Search is 4.9.0 .
We tried with localization on and off in the past but the user kept seeing this "GTSet.." .

**Reply by kasperbolarsen** [reply]
Any chance you can upgrade to the current release, 4.13.1?
We are only able to support the most recent release

**Reply by SiebeAmexio** [reply]
Hello, Sorry for the delay in responding. I will check if there is a possibility to upgrade.
